src/pages/ReviewPage.test.tsx

```diff
@@ -14,6 +14,9 @@ const contest1candidate0 = contest1.candidates[0]
 
 import ReviewPage from './ReviewPage'
 
+// a regexp to test for DOM IDs that start with a digit, which is a no-no.
+const badDOMIdRegexp = new RegExp('id="[0-9]')
```

Comment:
(nit-ish… but I think this is important to balance the volume of comments in the code.)

Naming is hard… but important:
- If the variable name is updated to add clarity such that comment is unnecessary, then we don't have to maintain comment in addition to variable.
- This line simply defines a pattern, there is no comparison here or anything "bad", thus the comment and var name do not describe what is happening.

I suggest moving line 18 down to where it is used and renaming to:

```
const idValueStartsWithNumberRegex = new RegExp('id="[0-9]')
expect(idValueStartsWithNumberRegex.test(container.outerHTML)).toBeFalsy()
```

Note: this all may be moot for this instance given comment below.

Reply:
I like the more semantic variable name. Good call.